docs/full-mesh.md

```diff
@@ -66,7 +66,7 @@ Then, create or update KnativeServing with the following options.
 
 ```shell
 cat <<EOF | oc apply -f -
-apiVersion: operator.knative.dev/v1alpha1
```

Comment:
Do we have a 1.26 ticket for updating docs?

Reply:
Created SRVCOM-2105 for this.